mue/README.md

```diff
@@ -0,0 +1,14 @@
+# Minimum Useless Engine
+
+A "minimum useless engine" for OpenSSL based on [this tutorial][tutorial].
```

Comment:
Broken markdown link?

Reply:
Link for `tutorial` is specified at bottom; did this way to follow how links were formatted in top-level README.